src/components/KeyboardShortcutsModal.js

```diff
@@ -32,57 +32,25 @@ const defaultProps = {
     isShortcutsModalOpen: false,
 };
 
-class KeyboardShortcutsModal extends React.Component {
-    componentDidMount() {
-        this.subscribedOpenModalShortcuts = [];
-
-        const openShortcutModalConfig = CONST.KEYBOARD_SHORTCUTS.SHORTCUT_MODAL;
-        this.unsubscribeShortcutModal = KeyboardShortcut.subscribe(
-            openShortcutModalConfig.shortcutKey,
-            () => {
-                if (this.props.isShortcutsModalOpen) {
-                    return;
-                }
-
-                ModalActions.close();
-                KeyboardShortcutsActions.showKeyboardShortcutModal();
-            },
-            openShortcutModalConfig.descriptionKey,
-            openShortcutModalConfig.modifiers,
-            true,
-        );
-
-        if (this.props.isShortcutsModalOpen) {
-            // The modal started open, which can happen if you reload the page when the modal is open.
-            this.subscribeOpenModalShortcuts();
-        }
-    }
-
-    componentDidUpdate(prevProps) {
-        if (!prevProps.isShortcutsModalOpen && this.props.isShortcutsModalOpen) {
-            this.subscribeOpenModalShortcuts();
-        } else if (prevProps.isShortcutsModalOpen && !this.props.isShortcutsModalOpen) {
-            // Modal is closing, remove keyboard shortcuts
-            this.unsubscribeOpenModalShortcuts();
-        }
-    }
-
-    componentWillUnmount() {
-        if (this.unsubscribeShortcutModal) {
-            this.unsubscribeShortcutModal();
-        }
-        this.unsubscribeOpenModalShortcuts();
-    }
+function KeyboardShortcutsModal({isShortcutsModalOpen = false, isSmallScreenWidth, translate}) {
+    const subscribedOpenModalShortcuts = useRef([]);
+    const closeShortcutEscapeModalConfig = CONST.KEYBOARD_SHORTCUTS.ESCAPE;
+    const closeShortcutEnterModalConfig = CONST.KEYBOARD_SHORTCUTS.ENTER;
+    const arrowUpConfig = CONST.KEYBOARD_SHORTCUTS.ARROW_UP;
+    const arrowDownConfig = CONST.KEYBOARD_SHORTCUTS.ARROW_DOWN;
+    const openShortcutModalConfig = CONST.KEYBOARD_SHORTCUTS.SHORTCUT_MODAL;
```

Comment:
i mean that you can place all this constants before function KeyboardShortcutsModal({isShortcutsModalOpen = false, isSmallScreenWidth, translate}) {
for example on line 34 - no needs to re-create them on every re-render of the component :-)